components/eam/src/chemistry/mozart/mo_neu_wetdep.F90

```diff
@@ -177,17 +179,19 @@ subroutine neu_wetdep_init
   do m=1,gas_wetdep_cnt
     call addfld     ('DTWR_'//trim(gas_wetdep_list(m)),(/ 'lev' /), 'A','kg/kg/s','wet removal Neu scheme tendency')
     call addfld     ('WD_'//trim(gas_wetdep_list(m)),horiz_only, 'A','kg/m2/s','vertical integrated wet deposition flux')
+    call addfld     ('HEFF_'//trim(gas_wetdep_list(m)),(/ 'lev' /), 'A','M/atm','Effective Henrys Law coeff.')
```

Comment:
Which ChemDyg metric requires the Henrys Law coefficient?

Reply:
It did not change compared to the previous code. It is only order different.